client/packages/android/capacitor.config.ts

```diff
@@ -23,6 +23,7 @@ const config: CapacitorConfig = {
     hostname: 'should.notmatch.localhost',
     // Required to access discovery graphql on http
     cleartext: true,
+    androidScheme: 'https',
```

Comment:
Would this mean that we `http` hot reload form local server (I guess can always run local https ?)

Reply:
er.. haven't tried. it might do yep.